src/app/app.component.ts

```diff
@@ -195,35 +203,83 @@ export class AppComponent implements AfterViewInit {
 
   async initializeTreeAsync() {
     this.ref.detectChanges();
-    await this.lfFolderBrowserElement?.nativeElement.initAsync({
-      treeService: this.lfRepoTreeService
-    });
+    let defaultNode = {
+      id: '168158',
+      isContainer: true,
```

Comment:
don't set a default node
also, same comment as on other PRs, should we just store the path instead of the folder